modules/mobile/docs/openapi.yaml

```diff
@@ -1477,6 +1477,82 @@ paths:
       security:
         - Bearer: []
       summary: /v0/health/immunizations
+  /v0/health/labs_and_tests:
+    get:
+      description: >
+        List patient labs and tests. Each report contains a list of which tests were a part of the report, along with
+        links to get the results for those tests.
+      operationId: list_labs_and_tests
+      parameters:
```

Comment:
So I think this confirms that we do need to pass the params to the service. Also, these are query params, not path params, right?

Reply:
Ya I can change them to query